### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by BTWS2**

### CODE REVIEW

File: nl/guides/pycharm-plugin/README.md [hunk 1]
```diff
@@ -5,36 +5,38 @@ description: "De Dodona PyCharm plugin"
...
 
-## Een API token aanmaken
+## 1. Een API token aanmaken
 Voor je de plugin kan installeren moet je eerst een API token aanmaken op de Dodona website. Via zo'n token kan PyCharm in jouw naam indienen zonder dat je je wachtwoord moet delen. Om een token aan te maken volg je [dit stappenplan](/nl/guides/creating-an-api-token). Hou het aangemaakte token bij, je zal het straks eenmalig in de plugin moeten invoeren.
 
-## De plugin installeren
+## 2. De plugin installeren
 
-Je kan de plugin direct installeren vanuit PyCharm. Je opent hiervoor eerst op het `File`-menu en vervolgens op `Settings...`.
+Je kan de plugin direct installeren vanuit PyCharm. Je opent hiervoor eerst het `File`-menu en klik vervolgens op `Settings...`.
```
Comment (bmesuere) [inline nl/guides/pycharm-plugin/README.md:15]:
```suggestion
Je kan de plugin direct installeren vanuit PyCharm. Je opent hiervoor eerst het `File`-menu en klikt vervolgens op `Settings...`.
```

File: nl/guides/pycharm-plugin/README.md [hunk 2]
```diff
@@ -5,36 +5,38 @@ description: "De Dodona PyCharm plugin"
...
 
-## Een nieuwe oefening starten
+## 3. Een nieuwe oefening starten
 
 PyCharm moet natuurlijk weten bij welke oefening hij je oplossing moet indienen. De eenvoudigste manier om dit te doen is door rechts te klikken op een bestaande map of project in PyCharm. In het menu dat verschijnt kies je bij `New` voor `Dodona Exercise`.
 
 ![new exercise](./new_exercise.png)
 
-Bij het aanmaken van je eerste oefening zal PyCharm je om je API token vragen. Vervolgens krijg je normaal een lijstje van al je Dodona cursussen te zien. Kies hier achtereenvolgens voor de juiste cursus, oefeningenreeks en oefening. Er zal nu automatisch een bestand aangemaakt worden waarin je je oplossing kan plaatsen.
+Bij het aanmaken van je eerste oefening zal **PyCharm** je om je **API token vragen**. Plak hier de API token die je daarnet aangemaakt hebt in de eerste stap.
```
Comment (bmesuere) [inline nl/guides/pycharm-plugin/README.md:31]:
```suggestion
Bij het aanmaken van je eerste oefening zal PyCharm je om je API token vragen. Plak hier de API token die je daarnet aangemaakt hebt in de eerste stap.
```

File: nl/guides/students/courses/README.md [hunk 3]
```diff
@@ -72,31 +72,31 @@ Een oefeningenreeks bestaat uit een beschrijving, een aantal oefeningen en event
 
 Als een reeks een deadline heeft, dan worden enkel ingediende oplossingen van voor dat tijdstip in rekening gebracht. Na de deadline kan je dus nog steeds indienen, maar ze tellen mogelijks niet mee voor een eventuele evaluatie. Meer details hierover kan je vinden bij de sectie [indienstatus](#indienstatus).
 
-Onder de naam van een oefeningenreeks staat optioneel een beschrijving, met daaronder een oplijsting van alle oefeningen uit de reeks. De lijst toont voor elke oefening je indienstatus en eventueel de voortgang van de groep. Vóór elke oefening in de lijst staat ook een icoontje dat correspondeert met je indienstatus voor de oefening.
+Onder de naam van een oefeningenreeks staat optioneel een beschrijving, met daaronder een oplijsting van alle oefeningen uit de reeks. De lijst toont voor elke oefening je indienstatus en eventueel de voortgang van de groep. Vóór elke oefening in de lijst staat ook een icoontje dat overeenkomt met je indienstatus voor de oefening.
 
-::: tip Belangrijk
+::: tip Dezelfde oefening in meerdere cursussen/oefeningenreeksen
```
Comment (bmesuere) [inline nl/guides/students/courses/README.md:77]:
```suggestion
::: tip Belangrijk
```

File: nl/guides/students/courses/README.md [hunk 4]
```diff
@@ -72,31 +72,31 @@ Een oefeningenreeks bestaat uit een beschrijving, een aantal oefeningen en event
...
 Dezelfde oefening kan voorkomen in meerdere cursussen. Je indienstatus voor de oefening wordt hierbij niet overgenomen. Je moet dus de oefening binnen elke cursus afzonderlijk indienen.
 
-Dezelfde oefening kan ook voorkomen in meerdere oefeningenreeksen van eenzelfde cursus. Hier worden je ingediende opgelossingen wel in rekening genomen alle voorkomens van de oefening. Afhankelijk van de ingestelde deadlines van de reeksen kan de indienstatus wel verschillen. Bij een reeks met een deadline worden namelijk enkel oplossingen die voor de deadline werden ingediend meegeteld.
+Dezelfde oefening kan ook voorkomen in meerdere oefeningenreeksen van eenzelfde cursus. Hier worden je ingediende opgelossingen wel in rekening genomen. Afhankelijk van de ingestelde deadlines van de reeksen kan de indienstatus wel verschillen. Bij een reeks met een deadline worden namelijk enkel oplossingen die voor de deadline werden ingediend meegeteld.
```
Comment (bmesuere) [inline nl/guides/students/courses/README.md:80]:
```suggestion
Dezelfde oefening kan ook voorkomen in meerdere oefeningenreeksen van eenzelfde cursus. Hier worden je ingediende opgelossingen wel in rekening genomenbij alle voorkomens van de oefening. Afhankelijk van de ingestelde deadlines van de reeksen kan de indienstatus wel verschillen. Bij een reeks met een deadline worden namelijk enkel oplossingen die voor de deadline werden ingediend meegeteld.
```

File: nl/guides/students/courses/README.md [hunk 5]
```diff
@@ -107,7 +107,7 @@ Mogelijke weergaven van je indienstatus nadat de deadline verstreken is:
   | `deadline gemist` (rood) | ![image](../../../../images/course_exercise_status_icons/after_deadline.png) | als je laatst ingediende oplossing niet correct is |
   | `deadline gemist` (rood) | ![image](../../../../images/course_exercise_status_icons/after_deadline.png) | als je na de deadline een correcte oplossing hebt ingediend |
 
-::: tip Belangrijk
+::: tip Belangrijk: laatst ingediende oplossing vóór deadline telt
```
Comment (bmesuere) [inline nl/guides/students/courses/README.md:110]:
```suggestion
::: tip Belangrijk
```

File: nl/guides/students/exercises/README.md [hunk 6]
```diff
@@ -1,8 +1,12 @@
 ---
-title: Oefeningen oplossen
+title: "[nl] Oefeningen oplossen"
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:2]: It seems like you put the English verion here and the Dutch version in the `en` directory.

File: nl/guides/students/exercises/README.md [hunk 7]
```diff
@@ -37,16 +41,20 @@ Als je een actie aan het uitvoeren bent op een oefening dan verschijnt de naam v
...
+Op een oefeningpagina staat onder het paneel met de beschrijving van de oefening een tweede paneel waarmee je een oplossing kunt indienen voor de oefening. Klik hiervoor op de tab `Indienen`, als deze tab niet geselecteerd was, en plaats de broncode van je oplossing in de *code editor*. Klik daarna op de indienknop in de rechterbovenhoek van het paneel om je oplossing in te dienen. **Je mag zoveel indienen als je wil. Er wordt enkel rekening gehouden met het resultaat van jouw laatste oplossing**. Bij elke oplossing wordt [automatische feedback](#interpreteren-van-feedback) door de judge gegeven die je kan gebruiken om je oplossing te corrigeren of verder te verfijnen.
 
 ![image](./student.exercise_before_submit.png)
 
-::: tip Tip
+::: tip Tip: gebruik een IDE (met plugin/extensie)
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:48]:
```suggestion
::: tip Tip
```

File: nl/guides/students/exercises/README.md [hunk 8]
```diff
@@ -37,16 +41,20 @@ Als je een actie aan het uitvoeren bent op een oefening dan verschijnt de naam v
...
+
+Bovendien zijn is er een plugin voorzien voor de [**JetBrains IDE**](jetbrains.com) ([IntelliJ IDEA](https://www.jetbrains.com/idea/), [PyCharm](https://www.jetbrains.com/pycharm/), [WebStorm](https://www.jetbrains.com/webstorm/specials/webstorm/webstorm.html), …) en een extensie voor [**Visual Studio Code**](https://code.visualstudio.com/). Programmeurs die met die IDE's werken kunnen hun oplossingen rechtstreeks in Dodona indienen met behulp van die tool. Zonder die tool moet je code kopiëren en plakken in het indieningstekstvak op Dodona en op de oranje cirkel te klikken. Hier vind je instructies om de plugin te installeren en te gebruiken:
+* [PyCharm plugin](../../pycharm-plugin/)  
+* [VS Code extensie](../../vs-code-extension/)
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:54]:
```suggestion
Bovendien zijn is er een plugin voorzien voor de JetBrains IDE's zoals [IntelliJ](https://www.jetbrains.com/idea/), [PyCharm](https://www.jetbrains.com/pycharm/), en [WebStorm](https://www.jetbrains.com/webstorm/specials/webstorm/webstorm.html). Ook voor [**Visual Studio Code**](https://code.visualstudio.com/) is een extensie voorzien. Programmeurs die met die IDE's werken kunnen hun oplossingen rechtstreeks in Dodona indienen met behulp van die tool. Zonder die tool moet je code kopiëren en plakken in het indieningstekstvak op Dodona en op de oranje cirkel te klikken. Instructies vind je [hier voor PyCharm](../../pycharm-plugin/) en [hier voor VS Code](../../vs-code-extension/).
```

File: nl/guides/students/exercises/README.md [hunk 9]
```diff
@@ -80,7 +88,7 @@ Je kunt een oplossing selecteren door in een oplossingenoverzicht op het pijltje
 
 ## Interpreteren van feedback
 
-Op de feedbackpagina staat gedetailleerde **feedback** over een oplossing die je ingediend hebt voor een oefening. Zo snel mogelijk na het indienen wordt de oplossing automatisch beoordeeld door een judge die aan de oefening gekoppeld is. Als motivatie van zijn beoordeling voorziet de judge gedetailleerde feedback over de oplossing.
+Op de feedbackpagina staat gedetailleerde **feedback** over een oplossing die je ingediend hebt voor een oefening. Zo snel mogelijk na het indienen wordt de oplossing automatisch beoordeeld door een judge die aan de oefening gekoppeld is. Als motivatie van zijn beoordeling voorziet de judge gedetailleerde feedback over de oplossing. De automatische feedback wordt opgesteld door de verwachte uitvoer en de uitvoer die gegenereerd werd aan de hand van de ingediende oplossing te vergelijken.
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:91]:
```suggestion
Op de feedbackpagina staat gedetailleerde **feedback** over een oplossing die je ingediend hebt voor een oefening. Zo snel mogelijk na het indienen wordt de oplossing automatisch beoordeeld door een judge die aan de oefening gekoppeld is. Als motivatie van zijn beoordeling voorziet de judge gedetailleerde feedback over de oplossing.
```
Dit is namelijk niet altijd zo.

File: nl/guides/students/login-and-settings/README.md [hunk 10]
```diff
@@ -17,7 +17,7 @@ Je kan Dodona gebruiken in het Nederlands of het Engels. Gebruik hiervoor het ta
 
 Als je een taal selecteert terwijl je aangemeld bent dan wordt die ook als voorkeurstaal ingesteld in je [gebruikersprofiel](https://dodona.ugent.be/profile). Op die manier wordt bij aanmelden automatisch je voorkeurstaal geselecteerd.
 
-::: tip Tip
+::: tip Wordt alles vertaald?
```
Comment (bmesuere) [inline nl/guides/students/login-and-settings/README.md:20]:
```suggestion
::: tip Tip
```

File: nl/guides/students/login-and-settings/README.md [hunk 11]
```diff
@@ -65,16 +65,18 @@ Op de profielpagina staan je persoonsgegevens (voornaam, familienaam, gebruikers
...
 
 ![image](./student.edit_timezone.png)
 
+::: tip Tijdzone verkeerd ingesteld?
 Als de tijdzone uit je gebruikersprofiel niet overeenkomt met de lokale tijdzone zoals die door je browser gedecteerd wordt, dan krijg je bovenaan elke pagina een mededeling te zien die erop wijst dat alle tijdsaanduidingen op het platform niet overeenkomen met de lokale tijdzone.
 
 ![image](./student.wrong_timezone.png)
 
 Je kunt deze mededeling enkel weghalen door in je gebruikersprofiel de tijdzone in te stellen op de lokale tijdzone. Merk op dat de mededeling een link bevat waarmee je rechtstreeks naar je profielpagina kunt navigeren.
+:::
```
Comment (bmesuere) [inline nl/guides/students/login-and-settings/README.md:82]:
```suggestion
Als de tijdzone uit je gebruikersprofiel niet overeenkomt met de lokale tijdzone zoals die door je browser gedecteerd wordt, dan krijg je bovenaan elke pagina een mededeling te zien die erop wijst dat alle tijdsaanduidingen op het platform niet overeenkomen met de lokale tijdzone.

![image](./student.wrong_timezone.png)

Je kunt deze mededeling enkel weghalen door in je gebruikersprofiel de tijdzone in te stellen op de lokale tijdzone. Merk op dat de mededeling een link bevat waarmee je rechtstreeks naar je profielpagina kunt navigeren.
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 12]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
+* `Beschrijving`: optioneel stuk dat bovenaan de cursuspagina wordt weergegeven. Voor het opmaken van de beschrijving kan je gebruikmaken van [Markdown](https://en.wikipedia.org/wiki/Markdown). Dodona maakt voor de weergave van Markdown gebruik van [kramdown](https://kramdown.gettalong.org) waardoor heel wat uitbreidingen van de standaard Markdown ondersteund worden. Voor meer uitleg over hoe deze formattering werkt, kan je [hier](/nl/references/exercise-description/) terecht.
+
+
+### Toegangsrechten
+
+
 * `Zichtbaarheid`: dit bepaalt of niet-geregistreerde gebruikers de cursus kunnen zien. Voor deze eigenschap kunnen de volgende waarden ingesteld worden: 
 
-  * `Zichtbaar`: alle gebruikers zien de cursus in het cursusoverzicht. Ze kunnen ook naar de cursuspagina navigeren en zich daar eventueel voor de cursus registreren.
+  * `Iedereen`: het cursusoverzicht is voor alle gebruikers **zichtbaar**. Ze kunnen ook naar de cursuspagina navigeren, alle zichtbare inhoud bekijken en zich daar eventueel voor de cursus registreren.
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:54]:
```suggestion
  * `Iedereen`: de cursus is voor alle gebruikers **zichtbaar** in de lijst van beschikbare cursussen. Ze kunnen ook naar de cursuspagina navigeren, alle zichtbare inhoud bekijken en zich daar eventueel voor de cursus registreren.
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 13]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
 
+* `Onderwijsinstelling`: een onderwijsinstelling selecteren kan gebruikers helpen om je cursus te vinden en kan ook gebruikt worden om de zichtbaarheid van en registraties voor deze cursus te beperken.
+
+* `Beschrijving`: optioneel stuk dat bovenaan de cursuspagina wordt weergegeven. Voor het opmaken van de beschrijving kan je gebruikmaken van [Markdown](https://en.wikipedia.org/wiki/Markdown). Dodona maakt voor de weergave van Markdown gebruik van [kramdown](https://kramdown.gettalong.org) waardoor heel wat uitbreidingen van de standaard Markdown ondersteund worden. Voor meer uitleg over hoe deze formattering werkt, kan je [hier](/nl/references/exercise-description/) terecht.
+
+
+### Toegangsrechten
+
+
 * `Zichtbaarheid`: dit bepaalt of niet-geregistreerde gebruikers de cursus kunnen zien. Voor deze eigenschap kunnen de volgende waarden ingesteld worden:
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:52]:
```suggestion
* `Voor wie is deze cursus zichtbaar`: dit bepaalt of niet-geregistreerde gebruikers de cursus kunnen zien. Voor deze eigenschap kunnen de volgende waarden ingesteld worden: 
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 14]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
 
-  * `Verborgen`: alleen cursusbeheerders zien de cursus in het cursusoverzicht en op hun startpagina. Er staat een icoontje bij om hen er op te wijzen dat andere gebruikers de cursus daar niet kunnen zien. Dit icoontje is ook te vinden op de cursuspagina zelf. Enkel geregistreerde gebruikers kunnen naar deze pagina navigeren. Andere gebruikers kunnen zich enkel voor de cursus registreren als ze een rechtstreekse registratielink gebruiken.
+  * `Enkel gebruikers van de ingestelde onderwijsinstelling`: de cursus is enkel toegankelijk voor gebruikers van de ingestelde onderwijsinstelling (zie `Onderwijsinstelling` hierboven) en een directe link (zie volgend item).
+
+  * `Enkel geregistreerde gebruikers`: alleen cursusbeheerders zien de cursus in het cursusoverzicht en op hun startpagina. Er staat een icoontje bij om hen er op te wijzen dat andere gebruikers de cursus daar niet kunnen zien. Dit icoontje is ook te vinden op de cursuspagina zelf. Enkel geregistreerde gebruikers kunnen naar deze pagina navigeren. Andere gebruikers kunnen zich enkel voor de cursus registreren als ze een rechtstreekse registratielink gebruiken. Voor alle andere gebruikers is de cursus **verborgen**.
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:58]:
```suggestion
  * `Enkel geregistreerde gebruikers`: alleen cursusbeheerders zien de cursus in het publieke cursusoverzicht. Er staat een icoontje bij om hen er op te wijzen dat andere gebruikers de cursus daar niet kunnen zien. Dit icoontje is ook te vinden op de cursuspagina zelf. Enkel geregistreerde gebruikers kunnen naar deze pagina navigeren. Andere gebruikers kunnen zich enkel voor de cursus registreren als ze een rechtstreekse registratielink gebruiken.
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 15]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
   
     ![verborgen cursus](./staff.course_hidden.png)
 
+* `Registratielink`: bij het aanmaken van een cursus wordt automatisch een registratielink gegenereerd als afschermingsmechanisme van verborgen cursussen. Zonder deze link kunnen niet-geregistreerde gebruikers de cursuspagina van een verborgen cursus niet zien en zich daar dus ook niet registreren. Je vindt deze registratielink op de cursus-bewerkenpagina. Klik op de kopieerknop naast de registratielink om de registratielink naar het klembord te kopiëren. 
+
+  ![registratielink](./staff.course_hidden_registration_link.png)
+
+  De registratielink heeft dus als voordeel dat gebruikers de cursus niet zelf moeten zoeken in het cursusoverzicht en dat ze meteen een verzoek krijgen om zich voor de cursus te registreren als ze dat nog niet gedaan hadden. Als cursusbeheerder krijg je de verantwoordelijkheid om de registratielink enkel te delen met niet-geregistreerde gebruikers die zich voor een verborgen cursus mogen registreren.
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:66]:
```suggestion
* `Toegangslink`: bij het aanmaken van een cursus wordt automatisch een toegangslink gegenereerd als afschermingsmechanisme van verborgen cursussen. Zonder deze link kunnen niet-geregistreerde gebruikers de cursuspagina van een verborgen cursus niet zien en zich daar dus ook niet registreren. Je vindt deze toegangslink op de cursus-bewerkenpagina. Klik op de kopieerknop naast de toegangslink om de toegangslink naar het klembord te kopiëren. 

  ![toegangslink](./staff.course_hidden_registration_link.png)

  De toegangslink heeft dus als voordeel dat gebruikers de cursus niet zelf moeten zoeken in het cursusoverzicht en dat ze meteen een verzoek krijgen om zich voor de cursus te registreren als ze dat nog niet gedaan hadden. Als cursusbeheerder krijg je de verantwoordelijkheid om de registratielink enkel te delen met niet-geregistreerde gebruikers die zich voor een verborgen cursus mogen registreren.  
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 16]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
   
     ![verborgen cursus](./staff.course_hidden.png)
 
+* `Registratielink`: bij het aanmaken van een cursus wordt automatisch een registratielink gegenereerd als afschermingsmechanisme van verborgen cursussen. Zonder deze link kunnen niet-geregistreerde gebruikers de cursuspagina van een verborgen cursus niet zien en zich daar dus ook niet registreren. Je vindt deze registratielink op de cursus-bewerkenpagina. Klik op de kopieerknop naast de registratielink om de registratielink naar het klembord te kopiëren. 
+
+  ![registratielink](./staff.course_hidden_registration_link.png)
+
+  De registratielink heeft dus als voordeel dat gebruikers de cursus niet zelf moeten zoeken in het cursusoverzicht en dat ze meteen een verzoek krijgen om zich voor de cursus te registreren als ze dat nog niet gedaan hadden. Als cursusbeheerder krijg je de verantwoordelijkheid om de registratielink enkel te delen met niet-geregistreerde gebruikers die zich voor een verborgen cursus mogen registreren.  
+
 * `Registratieprocedure`: deze eigenschap bepaalt of en hoe gebruikers zich voor de cursus kunnen registreren. Voor deze eigenschap kunnen de volgende waarden ingesteld worden:
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:68]:
```suggestion
* `Wie kan zich registreren voor deze cursus`: deze eigenschap bepaalt of en hoe gebruikers zich voor de cursus kunnen registreren. Voor deze eigenschap kunnen de volgende waarden ingesteld worden:
```

File: nl/guides/teachers/creating-a-course/README.md [hunk 17]
```diff
@@ -37,39 +37,49 @@ Finaal moet je nog enkele zaken over de cursus zelf, zoals de naam en lesgever,
...
-  * `Open`: gebruikers kunnen zich vrij voor de cursus registreren zonder expliciete goedkeuring van een cursusbeheerder.
+  * `Iedereen`: gebruikers kunnen zich vrij voor de cursus registreren zonder expliciete goedkeuring van een cursusbeheerder. De cursus staat dus **open** voor iedereen.
   
-  * `Gemodereerd`: gebruikers kunnen een registratieverzoek indienen voor de cursus maar zijn pas geregistreerd als een cursusbeheerder hun registratieverzoek heeft goedgekeurd.
+  * `Enkel gebruikers van {jouw_onderwijsinstelling}`: enkel gebruikers van je eigen onderwijsinstelling kunnen zich voor deze cursus registreren.
+
+  * `Niemand`: gebruikers kunnen zich niet zelf voor de cursus registreren. De registraties zijn dus **gesloten**.
 
-  * `Gesloten`: gebruikers kunnen zich niet zelf voor de cursus registreren.
+  Standaard verloopt de goedkeuring van de registratie **automatisch**. Er is ook een optie om het registratieproces **gemodereerd** te laten verlopen. Gebruikers kunnen dan een registratieverzoek indienen voor de cursus maar zijn pas geregistreerd als een cursusbeheerder hun registratieverzoek heeft goedgekeurd.
```
Comment (bmesuere) [inline nl/guides/teachers/creating-a-course/README.md:76]:
```suggestion
* Standaard verloopt de goedkeuring van de registratie **automatisch**. Er is ook een optie om het registratieproces **gemodereerd** te laten verlopen. Gebruikers kunnen dan een registratieverzoek indienen voor de cursus maar zijn pas geregistreerd als een cursusbeheerder hun registratieverzoek heeft goedgekeurd.
```

File: nl/guides/teachers/evaluate-series/README.md [hunk 18]
```diff
@@ -1,22 +1,21 @@
 ---
-title: "Een reeks evalueren"
+title: "Reeks evalueren"
```
Comment (bmesuere) [inline nl/guides/teachers/evaluate-series/README.md:2]:
```suggestion
title: "Een reeks evalueren"
```

File: nl/guides/teachers/getting-started/README.md [hunk 19]
```diff
@@ -18,7 +18,7 @@ Dodona wordt gratis aangeboden voor educatieve doeleinden.
 
 Aanmelden op Dodona kan via de Smartschool, Office 365 of G Suite account van je school. Dodona krijgt vervolgens je naam, emailadres, en naam van je school doorgestuurd van het gekozen loginsysteem. Op basis hiervan kunnen we je identificeren en maken we automatisch een Dodona-account aan de eerste keer dat je aanmeldt. Je wachtwoord wordt op geen enkel moment naar ons doorgestuurd.
 
-::: tip
+::: tip Meerdere inlogsystemen?
```
Comment (bmesuere) [inline nl/guides/teachers/getting-started/README.md:21]:
```suggestion
::: tip Tip
```

File: nl/references/exercise-config/README.md [hunk 20]
```diff
@@ -5,15 +5,15 @@ description: "Oefeningconfiguratie op Dodona"
...
 ## Configuratiebestandsstructuur voor oefeningen
 
-- **`type`**: Moet ingesteld worden op `exercise` voor oefeningen. De standaardwaarde indien afwezig is `exercise`
+- **`type`**: Moet ingesteld worden op `exercise` voor oefeningen. De standaardwaarde indien afwezig is `exercise`.
 - **`programming_language`** (string): de programmeertaal van de oefening, wordt gebruikt voor *syntax highlighting* en om de juiste bestandsextensie te bepalen
 - **`access`** (`public` of `private`): bepaalt wie deze oefening kan gebruiken
-  - public: elke lesgever op Dodona kan deze oefening gebruiker
-  - private: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
+  - `public`: elke lesgever op Dodona kan deze oefening gebruiker
+  - `private`: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
```
Comment (bmesuere) [inline nl/references/exercise-config/README.md:16]:
```suggestion
  - **`public`**: elke lesgever op Dodona kan deze oefening gebruiker
  - **`private`**: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
```

File: nl/references/exercise-config/README.md [hunk 21]
```diff
@@ -22,8 +22,8 @@ Dodona laat toe om de configuratie van een oefening of een leesactiviteit in te
   - **`handler`** (string, optioneel): de naam van de judge die gebruikt wordt voor de evaluatie. Standaard gebruik Dodona de judge die ingesteld is voor de repository.
   - **`image`** (string, optioneel): de naam van de docker image die gebruikt wordt voor de evaluatie. Standaard gebruikt Dodona de image die ingesteld is voor de judge.
   - **`time_limit`** (integer, optioneel): de tijd in seconden waarna de evaluatie van een oefening stopgezet wordt. Standaard is dit 42 seconden
-  - **`memory_limit`** (integer, optioneel): de hoeveelheid geheugen in bytes die gebruikt kan worden bij het uitvoeren van de eevaluatie. Standaard is dit ingesteld op 100M.
-  - **`network_enabled`** (boolean, optioneel): ingesteld op `true` als toegang tot het internet toegelaten is. Standaard staat deze waarde op `false`.
+  - **`memory_limit`** (integer, optioneel): de hoeveelheid geheugen in bytes die gebruikt kan worden bij het uitvoeren van de evaluatie. Standaard is dit ingesteld op 100M.
+  - **`network_enabled`** (**`false`** of `true`) (boolean, optioneel): ingesteld op `true` als toegang tot het internet toegelaten is. Standaard staat deze waarde op `false`.
```
Comment (bmesuere) [inline nl/references/exercise-config/README.md:26]:
```suggestion
  - **`network_enabled`** (boolean, optioneel): ingesteld op `true` als toegang tot het internet toegelaten is. Standaard staat deze waarde op `false`.
```

File: nl/references/exercise-config/README.md [hunk 22]
```diff
@@ -33,16 +33,16 @@ De structuur voor een leesactiviteit is identiek aan deze van een oefening. Er z
 
 - **`type`**: Moet ingesteld worden op `content` voor leesctiviteiten.
 - **`access`** (`public` of `private`): bepaalt wie deze oefening kan gebruiken
-  - public: elke lesgever op Dodona kan deze oefening gebruiker
-  - private: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
+  - `public`: elke lesgever op Dodona kan deze oefening gebruiker
+  - `private`: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
```
Comment (bmesuere) [inline nl/references/exercise-config/README.md:37]:
```suggestion
  - **`public`**: elke lesgever op Dodona kan deze oefening gebruiker
  - **`private`**: enkel lesgevers met expliciete toestemming mogen deze oefening gebruiken
```

File: nl/references/exercise-directory-structure/README.md [hunk 23]
```diff
@@ -5,27 +5,32 @@ description: "Oefeningmap-structuur Dodona"
...
+- **Een optioneel `readme.md`-, `readme.en.md`- en/of `readme.nl.md`-bestand:** De inhoud van deze bestanden zal getoond worden op de oefening-infopagina. Deze bestanden hebben als doel extra informatie geven aan leerkrachten die deze oefening misschien willen gebruiken in een cursus. Als er een bestand beschikbaar is in de taal van de gebruiker (`readme.<taal>.md`), dan zal deze getoond worden in plaats van het generieke `readme.md`-bestand. Dit is nuttig omdat `readme.md` getoond wordt door GitHub in de oefeningmap. We suggereren om een `readme.md`-bestand te maken in de taal van je doelpubliek en dit optioneeel te vertalen door `readme.nl.md` of `readme.en.md` te voorzien. 
+- **Een `description`-map**: deze map bevat de volgende bestanden die de oefening beschrijven:
   - **Een `description.en.md`- en/of `description.nl.md`-bestand**: deze bestanden bevatten de Engelse en/of Nederlandse beschrijving van de oefening.
 
+::: tip Voorbeeldcursus en voorbeeldoefeningen
```
Comment (bmesuere) [inline nl/references/exercise-directory-structure/README.md:15]:
```suggestion
::: tip Tip
```

File: nl/references/exercise-directory-structure/README.md [hunk 24]
```diff
@@ -5,27 +5,32 @@ description: "Oefeningmap-structuur Dodona"
...
-- **Een optionele `media`-folder**: deze folder bevat statische bestanden zoals afbeeldingen die gebruikt worden in de oefeningbeschrijving.
-- **Een optionele `boilerplate`-folder**: deze folder bevat de bestanden `boilerplate.en`, `boilerplate.nl`, en/of `boilerplate`. De inhoud van deze bestanden worden automatisch ingeladen in het inzendingstekstveld van de gebruikers. Je kan deze bestanden gebruiken om startcode of structuur te voorzien voor de studenten.
-- **Een `evaluation`-folder**: de inhoud van deze folder wordt beschikbaar gesteld voor de judge en kan bijvoorbeeld bestanden met de testcode bevatten.
-- **Een optionele `workdir`-folder**: de inhoud van deze folder worddt beschikbaar gesteld tijdens het uitvoeren van de judge en kan bijvoorbeeld databestanden bevatten die nodig zijn tijdens het uitvoeren van de tests.
-- **Een optionele `solutions`-folder**: bestanden in deze folder zullen getoond worden op de oefening-informatiepagina als voorbeeldoplossing. Meerdere voorbeeldoplossingen zijn mogelijk, maar bestanden met een naam beginnend met 'solution' zullen vooraan staan.
+> Deze mappen zijn enkel relevant voor programmeeroefeningen.
```
Comment (bmesuere) [inline nl/references/exercise-directory-structure/README.md:22]:
```suggestion
> Deze mappen zijn enkel relevant voor programmeeroefeningen en niet voor leesactiviteiten.
```

File: nl/references/python-judge/README.md [hunk 25]
```diff
@@ -7,8 +7,12 @@ description: "Python judge"
...
 
--   The **`OutputJudge`** klasse implementeert een judge die de ingediende broncode evalueert gebaseerd op de output die naar `stdout` wordt geschreven gebaseerd op input die via `stdin` wordt ingelezen.
--   De **`DoctestJudge`** klasse implementeeert een judge die de ingediende broncode evalueert door er een serie *unit tests* op uit te voeren die beschreven worden aan de hand van een uitgebreide versie van het format van de Python `doctest` module.
+-   The **`OutputJudge`** klasse implementeert een judge die de ingediende broncode evalueert gebaseerd op de output die naar `stdout` wordt geschreven gebaseerd op input die via `stdin` wordt ingelezen. Deze judge is dus geschikt voor oefeningen die input vragen via `input()` en de resultaten uitprinten via `print()`.
+-   De **`DoctestJudge`** klasse implementeeert een judge die de ingediende broncode evalueert door er een serie *unit tests* op uit te voeren die beschreven worden aan de hand van een uitgebreide versie van het format van de Python `doctest` module. Deze judge is geschikt om Python **functies** te testen.
+
+::: tip Voorbeeldcursus en voorbeeldoefeningen
```
Comment (bmesuere) [inline nl/references/python-judge/README.md:13]:
```suggestion
::: tip Tip
```

**Review by bmesuere — CHANGES_REQUESTED** [review]
Thank you for your edits 🙇 . I've added a bunch of suggestions. Some general remarks:
- I prefer not to have extended titles for the callouts
- I didn't add suggestions for the Dutch pages in the `en` directory. You should copy the files from the `nl` dirs after applying the suggestions
- For one guide, the English and Dutch file was swapped

Related: your PR for the example exercises repo will be reviewed at a later time. We plan to substantially improve those, but because that repository is used in our tests, we must be careful about making changes.

**Comment by BTWS2** [comment]
@bmesuere  My intention for adding titles to callouts, is that it allows for reading diagonally more easily. The title is a summary of what the callout is about. If the user reads the title and it's not applicable, the user can skip that section.

**Comment by bmesuere** [comment]
> @bmesuere My intention for adding titles to callouts, is that it allows for reading diagonally more easily. The title is a summary of what the callout is about. If the user reads the title and it's not applicable, the user can skip that section.

I can see your point. Can we agree on making them as short as possible? e.g. "Example course and example exercises" -> "Examples"?

File: nl/guides/teachers/creating-a-course/README.md (hunk 15 above)
Comment (BTWS2) [reply nl/guides/teachers/creating-a-course/README.md]: Fixed in commit because comment is outdated.

**Review by bmesuere — CHANGES_REQUESTED** [review]
At least the English "creating a course" page still is the old version, maybe other pages as well.

**Comment by BTWS2** [comment]
@bmesuere Yes indeed, I'm in a meeting now. I will transfer the changes to the English files before the end of today.

**Review by bmesuere — APPROVED** [review]

**Pull request merged**